README.md

```diff
@@ -1,63 +1,99 @@
         - `type`: Claim of type `d3-device-type-assertion` ([DOCS](https://specs.manysecured.net/d3/D3%20claims/#assert-device-type), [EXAMPLE]()).
         - `vuln`: Claim of type `d3-device-type-vuln` ([DOCS](https://specs.manysecured.net/d3/D3%20claims/#assert-device-type-vulnerability), [EXAMPLE]()).
     - Example templates for each type are in the `./examples` folder.
     - The YAML file values must not be preceded by tabs. If you want to achieve a visual indent use space characters.
     - If you want to generate UUID/GUIDs for your yaml definitions, refer to the uuid helper script in the [Workflow section](#workflow) below.
-4. If you are running on your local machine (requires Python 3 and [Python Poetry](https://python-poetry.org/))
+4. If you are **running on your local machine** (requires Python 3 and [Python Poetry](https://python-poetry.org/))
     - Run `cd d3-scripts/ && poetry install` to install the packages
     - Run `poetry run d3build` to run the D3 compiler against the repo locally. Any issues will manifest as errors.
     - Run `poetry run d3export` to run the exporter locally. This generates CSVs in the same schema as the SQL tables used by ManySecured devices
-5. If you can't check locally or after your satisfied with your local changes, push your branch changes to github, and create a pull request.
-6. The Github Actions CI will run the tests on the library and if it passes you can merge the changes.
+5. If you can't check locally or after your satisfied with your local changes, **push your branch changes to github, and create a pull request**.
+6. **The Github Actions CI will run the tests on the library and if it passes you can merge the changes.**
 
 ## Project Details
 
-### Folder Structure
-- `./manufacturers`: Store for YAML D3 claims/behaviours
-- `./manufacturers_json`: (Auto generated) Store for the compiled D3 claims
-- `./D3DB`: (Auto generated) Location of the exported D3 claim CSV tables
+### Repo Folder Structure
+- `./manufacturers`: Store for YAML D3 claims
+- `./manufacturers/D3_CORE`: contains useful common claim definitions
+- `./manufacturers_json`: (Auto generated on local compile) Store for the compiled D3 claims, each claim will have all it's inheritance references resolved. The compiled claims are in the JSON format.
+- `./D3DB`: (Auto generated) Location of the exported D3 claim schemas (all branches) and CSV tables (on the `csv` branch)
 - `./examples`: example claim and behaviour definitions
-- `./schemas`: Folder containing the JSON schemas for each D3 claim
 - `./d3-scripts`: Folder containing the scripts for compiling the D3 YAML claims
-- `./d3-scripts/src/d3_scripts/schemas/`: Folder containing the JSON schemas used in the compilation process
+- `./d3-scripts/src/d3_scripts/schemas/`: Folder containing the JSON schemas used in the compilation process should you want to 
```

Comment:
I believe this extra trailing space is the reason why the `pre-commit` CI is failing.

```suggestion
- `./d3-scripts/src/d3_scripts/schemas/`: Folder containing the JSON schemas used in the compilation process should you want to
```